src/components/Reports/ProjectMemberTable/ProjectMemberTable.jsx

```diff
@@ -1,33 +1,132 @@
 import { Stub } from 'components/common/Stub';
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './ProjectMemberTable.css';
+import { Link } from 'react-router-dom';
 
-export const ProjectMemberTable = ({ projectMembers, skip, take }) => {
-  let memberList = [];
-  if (projectMembers.fetched) {
-    if (projectMembers.members.length > 0) {
-      memberList = projectMembers.members.slice(skip, skip + take).map((member, index) => (
-        <div className="project-member-table-row" id={'tr_' + member._id} key={member._id}>
-          <div>
-            <div>{skip + index + 1}</div>
-          </div>
-          <div>
-            {member.firstName} {member.lastName}
+export const ProjectMemberTable = ({ projectMembers, skip, take, handleMemberCount }) => {
+  const [allMemberList, setAllMemberList] = useState([]);
+  const [activeMemberList, setActiveMemberList] = useState([]);
+  const [memberFilter, setMemberFilter] = useState('active');
+  const { fetched, foundUsers, members } = projectMembers;
+
+  let activeMemberTable = [];
+  let allMemberTable = [];
+
+  useEffect(() => {
+    let memberList = [];
+    let activeList = [];
+    let currentActive = [];
+    if (fetched) {
```

Comment:
Looks like these three arrays are not used if `fetched` is falsy. You could move them inside the block after checking `fetched`. And they can be declared with `const` since they don't get reassigned.